adafruit_portalbase/network.py

```diff
@@ -386,6 +386,21 @@ def connect(self, max_attempts=10):
                         "'networks' must be a list/tuple of dicts of 'ssid' and 'password'"
                     )
 
+            self._wifi_credentials = list(
+                filter(
+                    lambda credentials: isinstance(credentials, (list, tuple))
```

Comment:
I think this needs to be testing for `dict` instead of `tuple` and `list`. `self._wifi_credentials` gets created here: https://github.com/adafruit/Adafruit_CircuitPython_PortalBase/blob/d26e2324de496761e0aa72abc30ba07cdce8814b/adafruit_portalbase/network.py#L373-L378 as a list with a dictionary inside of it. 

I tested this branch with a standard settings.toml file containing my WIFI network and it raised the `No wifi credentials provided` exception even though my credentials are in settings. I can see them in there if I print the value of `self._wifi_credentials` as well.

```suggestion
                    lambda credentials: isinstance(credentials, dict)
```